charts/hivemq-edge/tests/default/hivemq_edge_configmap-configuration_test.yaml

```diff
@@ -0,0 +1,45 @@
+suite: HiveMQ Edge - Configmap Config tests
+
+templates:
+  - configmap-configuration.yml
+release:
+  name: edge
+  namespace: hivemq-edge-namespace
+chart:
+  version: 0.1.1
```

Comment:
I would recommend to use fake versions here, so the tests are clearly independent of any real versions. The templates anyway must work with any valid version. For example, for the platform tests we use:
```
chart:
  version: 0.0.1
  appVersion: 1.0.0
```

This also makes it easier to check if mandatory version bumps have been applied correctly (partially done by our Gradle release tasks).

Reply:
Done.